ci-operator/config/openshift/rosa/openshift-rosa-master__images.yaml

```diff
@@ -15,7 +15,9 @@ build_root:
 images:
 - dockerfile_literal: |
     FROM registry.ci.openshift.org/openshift/release:golang-1.16 as builder
-    RUN go get -u github.com/openshift/rosa/cmd/rosa
+    WORKDIR /go/src/github.com/openshift/rosa
+    RUN go install ./cmd/rosa
```

Comment:
Is this actually pulling down `github.com/openshift/rosa`? This seems like `/go/src/github.com/openshift/rosa` would be an empty dir

Reply:
No. In Prow CI, it will build the pipeline:src image which clones the code under test FROM the root image. Here, I input `src` to replace `registry.ci.openshift.org/openshift/release:golang-1.16` to be the new builder, so actually, we do not need the step `WORKDIR /go/src/github.com/openshift/rosa` at here, the start path for the `src` image is /go/src/github.com/openshift/rosa. I keep this useless step here as a message.

Reply:
You can get the detail information from the build log: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/30879/rehearse-30879-pull-ci-openshift-rosa-master-images-images/1552606946631094272/artifacts/build-logs/rosa-aws-cli.log